webnn/resources/test_data/instance_normalization.json

```diff
@@ -0,0 +1,519 @@
+{
+  "tests": [
+    {
+      "name": "instanceNormalization float32 4D tensor default options",
+      "inputs": {
+        "input": {
+          "shape": [2, 3, 2, 2],
+          "data": [
+            -97.949951171875,
+            29.44037628173828,
+            -73.92131042480469,
+            -38.11185836791992,
+            41.33772659301758,
+            -59.77853012084961,
+            -74.66901397705078,
+            -68.16508483886719,
+            35.82481384277344,
+            -6.948329448699951,
+            54.42462158203125,
+            47.53074645996094,
+            66.93562316894531,
+            76.74034881591797,
+            5.6758809089660645,
+            25.68659210205078,
+            37.37651062011719,
+            56.252689361572266,
+            -16.574905395507812,
+            42.949893951416016,
+            73.8739242553711,
+            -99.00035095214844,
+            -33.11322784423828,
+            -17.380685806274414
+          ],
+          "type": "float32"
+        }
+      },
+      "expected": {
+        "name": "output",
+        "shape": [2, 3, 2, 2],
+        "data": [
+          -1.0995290279388428,
+          1.5525832176208496,
+          -0.5992818474769592,
+          0.14622758328914642,
+          1.72129487991333,
+          -0.41020718216896057,
+          -0.7240943908691406,
+          -0.586993396282196,
+          0.13073226809501648,
+          -1.6633318662643433,
+          0.9108771681785583,
+          0.6217224597930908,
+          0.7947131395339966,
+          1.1309205293655396,
+          -1.3059037923812866,
+          -0.6197298169136047,
+          0.2657700479030609,
+          0.9459608793258667,
+          -1.6783342361450195,
+          0.46660327911376953,
+          1.5037200450897217,
+          -1.2981476783752441,
+          -0.2302791178226471,
+          0.024706769734621048
+        ],
+        "type": "float32"
+      }
+    },
+    {
+      "name": "instanceNormalization float32 4D tensor options.scale",
```

Comment:
Here's similar issue "NotSupportedError: DirectML: instanceNormalization: The scale and bias must be both given or not given." like **layerNormalization** .